### ISSUE TITLE
separate resources and the gaiasky program files

### ISSUE THREAD

**Issue by thometal** [langurmonkey/gaiasky#291]
Please separate the gaiasky program files and the resource files.

First thank you for the great program.

Optimal solution is in my option: To have gaiasky program files installed in the system directory. And resource files like dr2 catalog in some custom manageable location, select able in the menu. This has the favor the not download 700mb+ for nothing if selecting a different catalog. Also the resource files should be provided in the ".tar.xz" dataformat to save at least 12% of size in comparison of ".tar.gz" that would be over 8gb(estimate) for the extra large catalog.

**Comment by langurmonkey** [comment]
While the default settings install the catalog files in the same folder as the application, they can be moved easily by editing the `catalog-*.json` files in `gaiasky/assets/data`. A possible solution would be to offer a catalogless download option where the user manages the catalogs and their locations.
Regarding the `gz` vs `xz` issue, yes, compression rates are better with `xz`, but `gz` is much more common and available everywhere. Also, it is faster compressing and uncompressing. Of course, a case can be made on whether the shorter download time makes up for the longer decompression (it probably does). Will look into this.

**Comment by thometal** [comment]
I think the decompression usage is not relevant because the cpu requirements of the tool are much much higher. 
Also xz can be decompressed in parallel if there is more than one block.
Also xz is supported by nearly every linux distribution, now days.

**Comment by langurmonkey** [comment]
Branch [datasets-dl](/langurmonkey/gaiasky/tree/datasets-dl), which is already merged into master, takes care of the automatic downloading of the default catalog if none is found. After some testing I'll release it along with the next version of Gaia Sky.
Catalogs will still be gzipped thoguh :(

**Comment by langurmonkey** [comment]
Related to #303
